www/assets/de.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@mamahler: I'm not a fan of naming variables like that. If you read code without knowing the complete context, there is no way to guess what `_tr` is supposed to mean. Naming it `_translations` makes it obvious what this variables purpose is and I think the effort of writing out `_translations` is manageable, especially with autocomplete ;)